Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -341,6 +337,9 @@ class SignalAttachment: NSObject {
                 hasNonTextUTIType = true
             }
         }
+        if pasteboardUTISet.contains(kUTTypeURL as String) {
```

Comment:
maybe a comment about why this ordering is critical so we don't lose this knowledge in a "harmless" refactor.